### PR TITLE
Update what's new for PR #15512

### PR DESCRIPTION
**Pull request by codeofdusk**
### Link to issue number:
None

### Summary of the issue:
#15512 wasn't mentioned in the changelog for 2024.1.

### Description of how this pull request fixes the issue:
Update what's new accordingly.

### Testing strategy:
N/A

### Known issues with pull request:
None known

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -15,6 +15,7 @@ What's New in NVDA
 == Changes ==
 - NVDA no longer supports Windows 7 and Windows 8.
 Windows 8.1 is the minimum Windows version supported. (#15544)
+- The audio output device and ducking mode options have been removed from the "Select Synthesizer" dialog. They can be found in the audio settings panel which can be opened with ``NVDA+control+u``. (#15512)
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:18]: can this be moved to the bottom of the list? component updates should be second.

Also please split into new lines

```suggestion
- The audio output device and ducking mode options have been removed from the "Select Synthesizer" dialog.
They can be found in the audio settings panel which can be opened with ``NVDA+control+u``. (#15512)
```

**Comment by codeofdusk** [comment]
> can this be moved to the bottom of the list?

This should be displayed prominently, because it will break muscle memory for many long-term NVDA users and is probably more meaningful than dependency updates from an end user perspective.

**Comment by seanbudd** [comment]
@codeofdusk - there has been entire release, which will be out for 3+ months before this change is made. This will not be merged as is.

**Comment by Simon818** [comment]
I had no idea about the transition, and I frequently read NVDA changes but apparently I missed this one. From an end user perspective it needs to be near the top or someone will miss it, do something to rearrange their sound devices (Windows update often does this on its own) and not be able to rely on muscle memory to fix it anymore, leading to a decidedly bad day. Changes that rearrange a critical part of the user interface and could leave someone stranded without speech do not belong at the bottom of the list.

**Comment by CyrilleB79** [comment]
In 2023.3's change log, "New features" is the first paragraph and the first point starts as follows:

> * Enhanced sound management:
>   * A new Audio Settings panel:
>     * This can be opened with NVDA+control+u. (#15497)
>     * An option in Audio settings to have the volume of NVDA sounds and beeps follow the volume setting of the voice you are using. (#1409)
>     * An option in Audio settings to separately configure the volume of NVDA sounds. (#1409, #15038)
>     * The settings to change audio output device and toggle audio ducking have been moved to the new Audio settings panel from the Select Synthesizer dialog. These options will be removed from the "select synthesizer" dialog in 2024.1. (#15486, #8711)
> (...)

That is, the information was in the first paragraph, first bullet, first white level 2 bullet, fourth square level 3 bullet. Thus quite near from the top.

People reading only the initial blurb may have missed it but the people who have read the full log should have seen it.

IMO if you feel that the information should be more visible, it should even appear as a warning in the blurb. Else, the current location seems enough to me and moving it in the same list won't help so much.

**Comment by seanbudd** [comment]
Note that the order of change log entries will be changed closer to the release as we re-arrange and group items.

As Cyrille raised, this should be in the release blurb if visibility is important.
This change was at the top of the last release change log entries and also advertised in [in process](https://www.nvaccess.org/post/in-process-19th-october-2023/).

**Comment by lukaszgo1** [comment]
I'd  say that release blurb is indeed a right place to mention the change. To address several points from the above discussion:
- Mention  in In-Process is irrelevant for not English speaking users, and therefore increases visibility of the change only marginally
- The fact that 2023.3 was a transition release does not increase visibility of the change either, user can learn about it only when reading change log, or when stumbling on the new controls in the audio settings panel
- The change was not something requested by users (it is even perceived by some as changing interface for change's sake), and therefore we should do everything possible to make it less painfull
- Finally it is worth discussing what release blurbs are for - in my opinion while we obviously want to advertise the new release there, it is as important to mention anything which can make its adoption difficult. Change of supported versions of Windows, or move of commonly used options surely fall into this category.

**Comment by seanbudd** [comment]
We should also keep the release blurb continuously up to date, rather than writing it when branching for beta, so changes like this can be included.

**Comment by seanbudd** [comment]
Closing in favour of #15846

**Pull request closed without merging**